### DISCUSSION TITLE
Populate CFBundleShortVersionString in app bundle

### DISCUSSION THREAD

**Discussion by kevinmcox** [cirruslabs/tart#920] [Ideas]
Would it be possible to start populating the `CFBundleShortVersionString` key in the application bundle's `Info.plist` for each build to match `tart --version`?

Doing so would allow macOS to display the version number of Tart in the Finder and would also be helpful for version comparison in tools like [Munki](https://github.com/munki/munki).

**Comment by fkorotkov** [comment]
Available as of [2.19.0](https://github.com/cirruslabs/tart/releases/tag/2.19.0)

**Comment by kevinmcox** [comment]
I appreciate the quick attention! Unfortunately I'm not seeing it set in 2.19.0.

**Comment by edigaryev** [comment]
Should be now fixed and available in Tart 2.19.2:

```
% defaults read /opt/homebrew/Cellar/tart/2.19.2/libexec/tart.app/Contents/Info.plist CFBundleShortVersionString
2.19.2
```

**Reply by kevinmcox** [reply]
Looks good, thanks!
